docker-compose.dev.yml

```diff
@@ -7,8 +7,8 @@ services:
     restart: "no"
   dispatcher:
     restart: "no"
-  frontend:
```

Comment:
**question:** Is that change made on purpose?

Reply:
ah it meant to be quick and dirty change not meant to be committed, gonna rollback